src/styles/main.scss

```diff
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

Comment:
Need to keep the correct order as in some cases the 'reset' file can rewrite some particular styles from other modules (files).